lib/phoenix/presence.ex

```diff
@@ -5,9 +5,15 @@ defmodule Phoenix.Presence do
   This behaviour provides presence features such as fetching
   presences for a given topic, as well as handling diffs of
   join and leave events as they occur in real-time. Using this
-  module defines a supervisor and allows the calling module to
-  implement the `Phoenix.Tracker` behaviour which starts a
-  tracker process to handle presence information.
+  module defines a supervisor and a module that implements the
+  `Phoenix.Tracker` behaviour that uses `Phoenix.PubSub` to
+  broadcast presence updates.
+
+  In case you want to use only a subset of the functionality
+  provided by `Phoenix.Presence`, such as tracking processes
+  but without broadcasting updates, we recommend you to look
+  at the `Phoenix.Tracker` functionality from the `phoenix_pubsub`
+  project.
```

Comment:
I have detailed more the information you wanted to add, let me know what you think.

Reply:
Great edits @josevalim. I went ahead and removed it from the guide. 

I did tweak the wording around a little bit on one phrase "recommend you to look". Some of my co-workers and I have been discussing the use of "to" verb lately and I've realized that removing it from my writing has helped with clarity.